server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateDataStreamService.java

```diff
@@ -222,6 +221,13 @@ static ClusterState createDataStream(MetadataCreateIndexService metadataCreateIn
         logger.info("adding data stream [{}] with write index [{}] and backing indices [{}]", dataStreamName,
```

Comment:
maybe enhance this log line with the fact that aliases were created as well (if template has aliases)?